builtin/mainmenu/content/contentdb.lua

```diff
@@ -131,6 +142,16 @@ local function start_install(package, reason)
 					end
 					conf:set("author",     package.author)
 					conf:set("release",    package.release)
+					if package.aliases then
+						local gameid_aliases = {}
+						for _, alias in ipairs(package.aliases) do
+							local alias_cut = alias:match("[^/]+$")
+							if alias_cut ~= package.name then
+								gameid_aliases[#gameid_aliases + 1] = alias_cut
+							end
+						end
+						conf:set("aliases", table.concat(gameid_aliases, ","))
```

Comment:
why do we need to set these here?

Reply:
Because they're used when loading a world.

Reply:
And also when even finding the worlds to use for a game.

Reply:
I mean: do games not come with aliases set already in their game.conf?

Reply:
I don't think so, surrounding code is assembling literally the full game.conf...?

Reply:
It depends on whether you want the ability to alias a package on ContentDB without needing to update the .zip.

This is already supported for author changes - and is fine as nothing in a package depends on the author name and having the ability to transfer packages between authors is handy and quite common

For game name changes, it's a choice. It probably makes sense for ContentDB to require a release with an alias before accepting the name change. However, aliases are per-package on ContentDB not per-release. So you could end up with an alias in the package json but not in the release .zip still

TL;DR: :shrug:

Reply:
Actually I suppose after a name change you'll need to archive all previous releases anyway so that the package still works. So requiring authors to change the name in .zip is fine

Reply:
I'm fine either way, but what should we do if more than one game sets the same alias? Or do we just not care?